tests/microos/enable_nested_virt.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Enable nested virtualization support
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+
+sub run {
+    my $cpus = { GenuineIntel => 'intel', AuthenticAMD => 'amd' };
+    select_console 'root-console';
+    my $type = script_output(q(awk '/vendor_id/ {print $3}' /proc/cpuinfo | head -n 1));
+    assert_script_run(sprintf('echo "options kvm_%s nested=Y" > /etc/modprobe.d/50-kvm_%s.conf', $cpus->{$type}, $cpus->{$type}));
+    transactional::process_reboot(trigger => 1);
```

Comment:
just to be consistent with the other files, why not importing the module above and making here the call directly?